src/vol_geom.c

```diff
@@ -459,7 +459,7 @@ bool vol_geom_read_audio_from_file( const char* vols_filename, vol_geom_info_t*
 
   f_ptr = fopen( vols_filename, "rb" );
   if ( !f_ptr ) { goto vgraff_fail; }
-  if ( 0 != fseeko( f_ptr, info_ptr->hdr.audio_start, SEEK_SET ) ) { goto vgraff_fail; }
+  if ( 0 != vol_geom_fseeko( f_ptr, info_ptr->hdr.audio_start, SEEK_SET ) ) { goto vgraff_fail; }
```

Comment:
this is the key change. `fseeko` is the posix version and doesnt work on windows.